diff --git a/apps/dokploy/esbuild.config.ts b/apps/dokploy/esbuild.config.ts
index 9e6469ca..b8e0e212 100644
--- a/apps/dokploy/esbuild.config.ts
+++ b/apps/dokploy/esbuild.config.ts
@@ -1,20 +1,5 @@
 import esbuild from "esbuild";
 
-import dotenv, { type DotenvParseOutput } from "dotenv";
-
-const result = dotenv.config({ path: ".env.production" });
-
-function prepareDefine(config: DotenvParseOutput | undefined) {
-	const define = {};
-	// @ts-ignore
-	for (const [key, value] of Object.entries(config)) {
-		// @ts-ignore
-		define[`process.env.${key}`] = JSON.stringify(value);
-	}
-	return define;
-}
-
-const define = prepareDefine(result.parsed);
 try {
 	esbuild
 		.build({
@@ -31,7 +16,6 @@ try {
 			sourcemap: true,
 			outdir: "dist",
 			tsconfig: "tsconfig.server.json",
-			define,
 			packages: "external",
 		})
 		.catch(() => {
