diff --git a/apps/dokploy/components/layouts/dashboard-layout.tsx b/apps/dokploy/components/layouts/dashboard-layout.tsx
index 222f69b9..677b1605 100644
--- a/apps/dokploy/components/layouts/dashboard-layout.tsx
+++ b/apps/dokploy/components/layouts/dashboard-layout.tsx
@@ -23,11 +23,6 @@ export const DashboardLayout = ({ children }: Props) => {
 	return (
 		<>
 			<Page>{children}</Page>
-			{isChatEnabled && (
-				<>
-					<HubSpotWidget />
-				</>
-			)}
 
 			{haveRootAccess === true && <ImpersonationBar />}
 		</>
diff --git a/apps/dokploy/esbuild.config.ts b/apps/dokploy/esbuild.config.ts
index 615b2bdc..1e74fa44 100644
--- a/apps/dokploy/esbuild.config.ts
+++ b/apps/dokploy/esbuild.config.ts
@@ -1,24 +1,5 @@
-import dotenv, { type DotenvParseOutput } from "dotenv";
 import esbuild from "esbuild";
 
-const result = dotenv.config({ path: ".env.production" });
-
-function prepareDefine(config: DotenvParseOutput | undefined) {
-	const define = {};
-	// @ts-ignore
-	for (const [key, value] of Object.entries(config)) {
-		// Skip DATABASE_URL to allow runtime environment variable override
-		if (key === "DATABASE_URL") {
-			continue;
-		}
-		// @ts-ignore
-		define[`process.env.${key}`] = JSON.stringify(value);
-	}
-	return define;
-}
-
-const define = prepareDefine(result.parsed);
-
 try {
 	esbuild
 		.build({
@@ -38,7 +19,6 @@ try {
 			sourcemap: true,
 			outdir: "dist",
 			tsconfig: "tsconfig.server.json",
-			define,
 			packages: "external",
 		})
 		.catch(() => {
diff --git a/packages/server/src/lib/auth.ts b/packages/server/src/lib/auth.ts
index 3dea8f48..699ac2eb 100644
--- a/packages/server/src/lib/auth.ts
+++ b/packages/server/src/lib/auth.ts
@@ -325,7 +325,7 @@ const { handler, api } = betterAuth({
 					const host =
 						process.env.NODE_ENV === "development"
 							? "http://localhost:3000"
-							: "https://app.dokploy.com";
+							: process.env.PUBLIC_URL;
 					const inviteLink = `${host}/invitation?token=${data.id}`;
 
 					await sendEmail({
diff --git a/packages/server/src/services/admin.ts b/packages/server/src/services/admin.ts
index 9bc18334..acd80732 100644
--- a/packages/server/src/services/admin.ts
+++ b/packages/server/src/services/admin.ts
@@ -106,7 +106,7 @@ export const removeUserById = async (userId: string) => {
 
 export const getDokployUrl = async () => {
 	if (IS_CLOUD) {
-		return "https://app.dokploy.com";
+		return process.env.PUBLIC_URL;
 	}
 	const settings = await getWebServerSettings();
 
